src/org/usfirst/frc/team467/robot/DriverStation.java

```diff
@@ -116,6 +116,10 @@ else if(newSpeed < previousSpeed - maxAcceleration) {
 			previousSpeed -= maxAcceleration;
 		}
 		
+		else {
+			previousSpeed = newSpeed;
+		}
+		
```

Comment:
I think that's right, but I also think you can make it simpler. How about this:
`{`
`  previousSpeed = Math.min(previousSpeed + maxAcceleration, newSpeed);`
`  return previousSpeed;`
`}`

Reply:
That only works if the speed is increasing. The current change is fine.

Reply:
Oops, Nathan pointed out my suggestion doesn't work for reverse. :-}